payments-core/src/main/java/com/stripe/android/model/PaymentIntent.kt

```diff
@@ -17,7 +17,9 @@ import java.util.regex.Pattern
  * - [PaymentIntents API Reference](https://stripe.com/docs/api/payment_intents)
  */
 @Parcelize
-data class PaymentIntent internal constructor(
+data class PaymentIntent
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+constructor(
```

Comment:
This makes me nervous, even though we have `RestrictTo`

Reply:
The worst part about this is, it's ONLY for testing 😭 But I needed to create this to make some tests in paymentsheet compile.

Reply:
Can we mock?

Reply:
I try to avoid mocks 🙈 But if you prefer that, I can change it!